models/platform-limits/src/business-units/builder.spec.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TBusinessUnitLimitsProjection } from './types';
+import * as BusinessUnitsPlatformLimits from './index';
+
+describe('building', () => {
+  it(
+    ...createBuilderSpec<
+      TBusinessUnitLimitsProjection,
+      TBusinessUnitLimitsProjection
+    >(
+      'default',
+      BusinessUnitsPlatformLimits.random(),
```

Comment:
Here we're testing the default values of the generator, which does not seem to provide too much value since all properties are initialized with `null`.
Perhaps we can use the `withLimit` preset so we make sure the model is instantiated with some actual data.

Reply:
done https://github.com/commercetools/test-data/pull/655/commits/617107cad9f4c095a0f686bf60262289367ca677